### ISSUE TITLE
System.InvalidTimeZoneException : "US/Pacific" was not recognized as a valid Windows time zone ID.

### ISSUE THREAD

**Issue by parekhkb** [mj1856/TimeZoneConverter#23]
We recently tried upgarding to v2.3.2 from v2.2.1. 
When we did, one of our test cases starting failing on Linux but passing on windows:

```c#
        [Fact]
        public void GetTimeZoneInfo_PassTimeZoneIds_ReturnsCorrectTimeZone()
        {
            void ShouldBeEqual(string winId, string ianaId)
            {
                var tzWindows = TZConvert.GetTimeZoneInfo(winId);
                var tzIana = TZConvert.GetTimeZoneInfo(ianaId);
                tzWindows.Should().Be(tzIana);
            }

            ShouldBeEqual("Eastern Standard Time", "America/New_York");
            ShouldBeEqual("Central Standard Time", "America/Chicago");
            ShouldBeEqual("Mountain Standard Time", "America/Denver");
            ShouldBeEqual("Pacific Standard Time", "America/Los_Angeles");
        }
```

```
Error Message:
 System.InvalidTimeZoneException : "US/Pacific" was not recognized as a valid Windows time zone ID.
Stack Trace:
   at TimeZoneConverter.TZConvert.WindowsToIana(String windowsTimeZoneId, String territoryCode)
at TimeZoneConverter.TZConvert.GetTimeZoneInfo(String windowsOrIanaTimeZoneId)
```

**Comment by mj1856** [comment]
Thanks for the bug report.  I'll look into this soon.

**Comment by mj1856** [comment]
This should be fixed now in version 2.4.0.

Basically the same issue as in #21 and #26.

Thanks.

**Comment by parekhkb** [comment]
Thanks that fixed it.
